src/Ci/SourceHut.php

```diff
@@ -0,0 +1,60 @@
+<?php declare(strict_types=1);
+
+namespace OndraM\CiDetector\Ci;
+
+use OndraM\CiDetector\CiDetector;
+use OndraM\CiDetector\Env;
+use OndraM\CiDetector\TrinaryLogic;
+
+class SourceHut extends AbstractCi
+{
+    public static function isDetected(Env $env): bool
+    {
+        return ($env->get('JOB_ID') !== false) && $env->get('JOB_URL') !== false;
```

Comment:
This really seems a bit too generic for the detection and may cause false positives ... If sr.ht does not have any specific variable on which the detection can be based (nor they are willing to add it), maybe we can check whether JOB_URL contains for example `sr.ht`? Its not optimal, but will be specific enough.

Reply:
Hi,

I agree, I'm concerned a future CI system could feasibly have `JOB_ID` and `JOB_URL` in the future. Unfortunately, those are basically the only env vars available to check for

> maybe we can check whether JOB_URL contains for example sr.ht

That would work for this specific case, but sr.ht is a public instance of open source software. Ideally if I run my own instance, ci-detector would still work. Perhaps we can just add support for builds.sr.ht for now

Reply:
Agreed, this sounds reasonable.